### DISCUSSION TITLE
Unable to locate package systemd-genie

### DISCUSSION THREAD

**Discussion by RavinRau** [arkane-systems/genie#193] [Q&A] [answered]
Hi I am kinda a newbie with wsl2 and Linux, I follow the step in the docs but when it comes to `sudo apt install -y systemd-genie` i am getting `E: Unable to locate package systemd-genie`

Am I doing something wrong? 

![image](https://user-images.githubusercontent.com/12477629/129176136-b73c90b7-7f8b-44a3-aae5-c9d0f74514a7.png)

Note: I am using 
WSL2, Ubuntu 20.04 LTS

**Comment by cerebrate** [comment, marked as answer]
I'm not seeing the repository for the genie packages in the list `apt update` produced. You need to follow the instructions here to add it before you can install genie:

https://arkane-systems.github.io/wsl-transdebian/

**Reply by telestrialOne** [reply]
Does this work for Ubuntu 22.04 Jammy Jellyfish

**Reply by cerebrate** [reply]
Maybe. I haven't tested it on that release, but there's probably no reason you couldn't install the 20.04 Ubuntu package and expect it to work.
